source/common/ssl/context_config_impl.cc

```diff
@@ -189,7 +189,10 @@ ServerContextConfigImpl::ServerContextConfigImpl(
       }()) {
   // TODO(PiotrSikora): Support multiple TLS certificates.
   if ((config.common_tls_context().tls_certificates().size() +
-       config.common_tls_context().tls_certificate_sds_secret_configs().size()) != 1) {
+       config.common_tls_context().tls_certificate_sds_secret_configs().size()) == 0) {
+    throw EnvoyException("Missing tls_certificates specification for server context");
```

Comment:
Nit: Because the certificates can come from different sources, it might be better to just say "No TLS certificates found for server context"